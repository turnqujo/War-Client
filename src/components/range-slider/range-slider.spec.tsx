import { shallow } from 'enzyme';
import React from 'react';
import RangeSlider from './range-slider';

describe('The Range control Component', () => {
  test('renders without crashing.', () => {
    shallow(<RangeSlider domain={{ start: 1, end: 100 }} initial={{ start: 1, end: 10 }} onUpdate={() => { }} />);
  });

  test('should match its snapshot.', () => {
    const wrapper = shallow(
      <RangeSlider domain={{ start: 1, end: 100 }} initial={{ start: 1, end: 10 }} onUpdate={() => { }} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
