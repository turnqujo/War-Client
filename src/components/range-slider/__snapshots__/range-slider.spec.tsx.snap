// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Range control Component should match its snapshot. 1`] = `ShallowWrapper {}`;
